### PR TITLE
Selected Bluetooth/MPSL samples: Support nRF54L10 and nRF54L05

### PR DESCRIPTION
**Pull request by rugeGerritsen**
These SoCs are similar to nRF54L15. No change to the sample is needed. This PR updates sample support for samples with codeowner set to @nrfconnect/ncs-dragoon.

Each of the samples has updated both integration_platform and platform_allow.

### CODE REVIEW

**Comment by rugeGerritsen** [comment]
@nrfconnect/ncs-dragoon-doc , does this need an entry in the changelog? I would assume that can be taken separately.

**Comment by Tschet1** [comment]
Should we also add support for l10 and l05 for the iso_combined_bis_and_cis sample? Or does that not fit?

**Review by Tschet1 — APPROVED** [review]

**Comment by rugeGerritsen** [comment]
> Should we also add support for l10 and l05 for the iso_combined_bis_and_cis sample? Or does that not fit?

Thanks for pointing out. I missed those.

**Comment by rugeGerritsen** [comment]
@nrfconnect/ncs-ci , requested a review from you to check if we want to go for the approach of adding all 54L to platform_allow and integration_platform

**Comment by rugeGerritsen** [comment]
Thanks for pointing out @wiba-nordic , will add it

File: doc/nrf/includes/sample_board_rows.txt [hunk 1]
```diff
@@ -222,6 +222,14 @@
 
 | Native Simulator | | native_sim | ``native_sim`` |
 
+.. nrf54l15dk_nrf54l05_cpuapp
+
+| :ref:`nRF54L15 DK (emulating nrf54l05)` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l05/cpuapp`` |
```
Comment (wiba-nordic) [inline doc/nrf/includes/sample_board_rows.txt:227]:
```suggestion
| :ref:`nRF54L15 DK (emulating nrf54l05) <ug_nrf54l>` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l05/cpuapp`` |
```

File: doc/nrf/includes/sample_board_rows.txt [hunk 2]
```diff
@@ -222,6 +222,14 @@
...
+
+.. nrf54l15dk_nrf54l10_cpuapp
+
+| :ref:`nRF54L15 DK (emulating nrf54l10)` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l10/cpuapp`` |
```
Comment (wiba-nordic) [inline doc/nrf/includes/sample_board_rows.txt:231]:
```suggestion
| :ref:`nRF54L15 DK (emulating nrf54l10) <ug_nrf54l>` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l10/cpuapp`` |
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/includes/sample_board_rows.txt [hunk 3]
```diff
@@ -222,6 +222,14 @@
 
 | Native Simulator | | native_sim | ``native_sim`` |
 
+.. nrf54l15dk_nrf54l05_cpuapp
+
+| :ref:`nRF54L15 DK (emulating nrf54l05)` <ug_nrf54l>`  | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l05/cpuapp`` |
```
Comment (wiba-nordic) [inline doc/nrf/includes/sample_board_rows.txt:227]:
```suggestion
| :ref:`nRF54L15 DK (emulating nrf54l05) <ug_nrf54l>`  | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l05/cpuapp`` |
```

File: doc/nrf/includes/sample_board_rows.txt [hunk 4]
```diff
@@ -222,6 +222,14 @@
...
+
+.. nrf54l15dk_nrf54l10_cpuapp
+
+| :ref:`nRF54L15 DK (emulating nrf54l10)` <ug_nrf54l>` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l10/cpuapp`` |
```
Comment (wiba-nordic) [inline doc/nrf/includes/sample_board_rows.txt:231]:
```suggestion
| :ref:`nRF54L15 DK (emulating nrf54l10) <ug_nrf54l>` | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l10/cpuapp`` |
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/includes/sample_board_rows.txt (hunk 3 above)
Comment (peknis) [reply doc/nrf/includes/sample_board_rows.txt]: emulating nRF54L05

File: doc/nrf/includes/sample_board_rows.txt (hunk 4 above)
Comment (peknis) [reply doc/nrf/includes/sample_board_rows.txt]: Rather: (emulating nRF54L10)

**Comment by wiba-nordic** [comment]
I'll wait for the build to finish and check one or two of the samples, but looks good.

**Comment by rugeGerritsen** [comment]
> I'll wait for the build to finish and check one or two of the samples, but looks good.

Note that we should update https://github.com/nrfconnect/sdk-zephyr/blob/main/boards/nordic/nrf54l15dk/doc/index.rst with a section about emulated targets like we do for 52. I believe that is better done separately

**Review by peknis — APPROVED** [review]

**Review by wiba-nordic — COMMENTED** [review]
Noticed we've got pages that actually mention the emulation, so changing the targets to those.

**Review by wiba-nordic — APPROVED** [review]

File: doc/nrf/includes/sample_board_rows.txt [hunk 5]
```diff
@@ -222,6 +222,14 @@
 
 | Native Simulator | | native_sim | ``native_sim`` |
 
+.. nrf54l15dk_nrf54l05_cpuapp
+
+| :ref:`nRF54L15 DK (emulating nRF54L05) <ug_nrf54l>`  | PCA10156 | :ref:`nrf54l15dk <zephyr:nrf54l15dk_nrf54l15>` | ``nrf54l15dk/nrf54l05/cpuapp`` |
```
[the remark this answers is not in the document]
Comment (wiba-nordic) [reply doc/nrf/includes/sample_board_rows.txt:227]: Ah, excellent. Then everything is good.

**Review by annwoj — APPROVED** [review]

**Comment by rugeGerritsen** [comment]
Talked to @jangalda-nsc  about adding more integration targets, and he said this was fine.

**Pull request merged**